README.md

```diff
@@ -1,59 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Fantasy Bookstore
 
-> One paragraph statement about the project.
+> "Fantasy Bookstore" is a website for Fantasy book fans. It is a bookstore which allows the user to interact with it in the following manner:
+  - Add Books to a list.
+  - Remove selected book.
+  - Use of api to store the given response.
+  By building this application, I will get my first experiance of React-Redux. 
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
-
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html, CSS, Javascript
+- React, Redux, Thunk
+- Webpack, Jest, Gitflow
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link]()
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
+To get a local copy up and running in your machine:
 
-### Install
+    1.Clone this repo or download the zip folder:
 
-### Usage
-
-### Run tests
-
-### Deployment
+      git clone https://github.com/Lockless404/dotaCapstone.git
 
+    2.Navigate to the location of the folder in your machine:
 
+      you@your-Pc-name:~$ cd <folder>
```

Comment:
Please add `npm install`, and `npm start` commands as these are necessary for installing and running the project locally.